realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -518,6 +560,82 @@ public void cancel() {
         }
     }
 
+    private class MakeOfferAsyncTask extends AsyncTask<String> {
```

Comment:
Seems plausible for an implementation of the Command pattern.  Perhaps it is our guidelines that should be updated, here?